content/en/tracing/trace_collection/library_injection_local.md

```diff
@@ -93,14 +95,21 @@ To select your pods for library injection, annotate them with the following, cor
 | Java       | `admission.datadoghq.com/java-lib.version: "<CONTAINER IMAGE TAG>"`   |
 | JavaScript | `admission.datadoghq.com/js-lib.version: "<CONTAINER IMAGE TAG>"`     |
 | Python     | `admission.datadoghq.com/python-lib.version: "<CONTAINER IMAGE TAG>"` |
+| .NET       | `admission.datadoghq.com/dotnet-lib.version: "<CONTAINER IMAGE TAG>"` |
+| Ruby       | `admission.datadoghq.com/ruby-lib.version: "<CONTAINER IMAGE TAG>"` |
 
 The available library versions are listed in each container registry, as well as in the tracer source repositories for each language:
-- [Java][13]
-- [Javascript][14]
-- [Python][15]
+- [Java][16]
+- [Javascript][17]
+- [Python][18]
+- [.NET][19]
+- [Ruby][20]
+  - Note: If the application container uses a musl-based Linux distribution (such as Alpine), you must specify a tag with the the `-musl` suffix for the pod annotation. For example, if you would like to use library version `v2.29.0`, specify container tag `v2.29.0-musl`.
 
 **Note**: If you already have an application instrumented using version X of the library, and then use library injection to instrument using version Y of the same tracer library, the tracer does not break. Rather, the library version loaded first is used. Because library injection happens at the admission controller level prior to runtime, it takes precedent over manually configured libraries.
 
+**Note**: For Ruby, only support instrumenting Ruby on Rails or Hanami application.
```

Comment:
```suggestion
**Note**: For Ruby, instrumenting only Ruby on Rails or Hanami applications is supported.
```

Reply:
Small note: Could this go in a sub-bullet, kind of like the .NET one? Otherwise the suggested copy looks good to me

Reply:
yeah, I'll update your branch with that